lib/dialogs/share.jsx

```diff
@@ -111,7 +112,7 @@ export default React.createClass( {
 		const { note } = this.props.params;
 		const data = note && note.data || {};
 		const isPublished = includes( data.systemTags, 'published' );
-		const publishURL = isPublished && data.publishURL !== '' && data.publishURL && `http://simp.ly/publish/${data.publishURL}` || null;
+		const publishURL = ! isEmpty( data.publishURL ) && `http://simp.ly/p/${data.publishURL}` || null;
```

Comment:
There another approach that may be more elegant too at the cost of a bit more mental work. What we're really wanting is a function that gives us a default value like `get( object, path, default )` except that we want it to use the default whenever the value is string-empty, a `getWith( object, path, default[ predicate, defaultValue ] )`

Iteration 1 - make a simple function, maybe the best option here

```js
const getPublishUrl = url => isEmpty( url ) ? null : `http://simp.ly/p/${url}`;

…

// good and good - now it's null if the string is empty
const publishURL = getPublishUrl( data.publishURL );
```

Iteration 2 - make our reusable getWith()

```js
const getWith = ( object, path, predicate, default ) => {
	const value = get( object, path );
	return predicate( value ) ? value : default;
};

…
const notEmpty = negate( isEmpty );
const publishURL = getWith( data, 'publishURL', notEmpty, null );
```